src/extensions/default/bramble/stylesheets/sidebarTheme.css

```diff
@@ -1,68 +1,250 @@
+@import url('https://fonts.googleapis.com/css?family=Open+Sans:400,600');
```

Comment:
We use it in Thimble, but it doesn't seem to be available to brackets (probably because it sits in an iframe). I can file a follow up issue to see if we are double-loading and what our options might be if we are.